cli/command/registry.go

```diff
@@ -178,7 +178,7 @@ func PromptUserForCredentials(ctx context.Context, cli Cli, argUser, argPassword
 			}
 		}()
 
-		argPassword, err = PromptForInput(ctx, cli.In(), cli.Out(), "Password: ")
+		argPassword, err = PromptForInput(ctx, cli.In(), cli.Out(), "Password/PAT: ")
```

Comment:
The PAT feels a bit awkward as a prompt. @thaJeztah @laurazard  any opinions on this?

Reply:
Hmm, I'm a bit unsure. I think it's nice to be explicit (we could even say `secret`!) but I think in general, password is what's expected by users, and other CLIs that also accept other types of secrets usually default to `password`.